live-examples/html-examples/forms/textarea.html

```diff
@@ -0,0 +1,40 @@
+<div class="textarea-example">
+    <div>
+        <label for="basic">Basic example:</label>
+        <textarea id="basic" name="basic">
+        </textarea>
+    </div>
+    <div>
+        <label for="disabled">Disabled example:</label>
+        <textarea id="disabled" name="disabled" disabled>
+Disabled textarea
+        </textarea>
+    </div>
+    <div>
+        <label for="readonly">Readonly example:</label>
+        <textarea id="readonly" name="readonly" readonly>
+This content is READONLY.
+        </textarea>
+    </div>
+    <div>
+        <label for="req-place">Required example with placeholder:</label>
+        <textarea id="req-place" name="req-place"
+                required placeholder="This is placeholder text">
+        </textarea>
+    </div>
+    <div>
+        <label for="sizing-wrapping">Sized example:</label>
+        <textarea id="sizing-wrapping" name="size"
+                  rows="5" cols="50" wrap="hard">
+Textarea with explicit sizing and wrap attribute to control wrapping. try values of hard, soft, and off in the wrap attribute to see how it affects the text wrapping
+        </textarea>
+    </div>
+    <div>
+        <label for="min-max">minlength/maxlength example:</label>
+        <textarea id="min-max" name="min-max"
+                  rows="4" cols="50"
+                  minlength="3" maxlength="45">
```

Comment:
minlength did not work for me here (Chrome 66, Firefox 61). At least, it let me delete everything in the box. Is that the intended behavior?